cli/internal/prune/prune.go

```diff
@@ -90,24 +92,57 @@ func (p *prune) prune(opts *opts) error {
 	if err != nil {
 		return errors.Wrap(err, "could not construct graph")
 	}
-	p.base.Logger.Trace("scope", "value", opts.scope)
-	target, scopeIsValid := ctx.PackageInfos[opts.scope]
-	if !scopeIsValid {
-		return errors.Errorf("invalid scope: package %v not found", opts.scope)
+
+	var pkgDepGraph *context.Context
```

Comment:
This section was copied directly from [run.go](https://github.com/vercel/turbo/blob/6a0228125cafbbceed53ff17a7b25259422df568/cli/internal/run/run.go#L194). I left out the daemon-related things, which seemed less applicable to `prune` that's typically run once in an ephemeral environment.

Reply:
The logic copied here should be unnecessary if we only need to allow for users to be able to prune down to multiple explicitly named packages.